models/migrations/migrations.go

```diff
@@ -554,6 +554,9 @@ var migrations = []Migration{
 	NewMigration("Add combined Index to issue_user.uid and issue_id", v1_22.AddCombinedIndexToIssueUser),
 	// v284 -> v285
 	NewMigration("Add ignore stale approval column on branch table", v1_22.AddIgnoreStaleApprovalsColumnToProtectedBranchTable),
+	// v285 -> v286
+	NewMigration("Add support for SHA256 git hash references", v1_22.ExpandHashReferencesToSha256),
+	NewMigration("Add ObjectFormatName to Repository table", v1_22.AddObjectFormatNameToRepository),
```

Comment:
Merge the two migration as one since it's not merged.

Reply:
https://github.com/go-gitea/gitea/pull/23894/commits/73c67e46229729e62c0c4d6eb6a2be9a56b1a2f3